src/bentoml/_internal/frameworks/pytorch.py

```diff
@@ -43,9 +44,19 @@ def get(tag_like: str | Tag) -> Model:
     return model
 
 
+class ModelOptions(PartialKwargsModelOptions):
+    fullgraph: bool = False
+    dynamic: bool = False
+    backend: t.Union[str, t.Callable[..., t.Any]] = "inductor"
+    mode: t.Optional[str] = None
+    options: t.Optional[t.Dict[str, t.Union[str, int, bool]]] = None
+    disable: bool = False
+
+
```

Comment:
First, let’s call this `PytorchOptions` and only when import to `bentoml.pytorch` we rename it to `ModelOptions`.

Second, I think maybe it’s better to have single `compile_kwargs` dict in `PytorchOptions` instead of polluting the name space of `PytorchOptions`. Maybe we will have only 2 entries:

- `enable_compile`
- `compile_kwargs`